### ISSUE TITLE
upload-debug-symbols-win.ps1 is not digitally signed

### ISSUE THREAD

**Issue by riot-pmaconi** [getsentry/sentry-unreal#275]
### Environment

How do you use Sentry?
Sentry SaaS

Which version of the SDK?
0.5.0

How did you install the package? (Git-URL, Assetstore)
Git-URL

Which version of Unreal?
5.1

Is this happening in Unreal (editor) or on a player like Android, iOS, Windows?
Editor

### Steps to Reproduce

1. Enable the Sentry plugin
2. Attempt to build the Unreal editor

### Expected Result

The build completes successfully.

### Actual Result

The build fails due to an unsigned PowerShell script. This seems to be the same issue as https://github.com/getsentry/sentry-unreal/issues/83, though that solution still appears to be in place and did not resolve it for me.

### Any logs or screenshots

```
ubt> [17/17] cmd Executing post build script (PostBuild-1.bat)
ubt> File \<project location\>\Sentry\Scripts\upload-debug-symbols-win.ps1 cannot be
ubt> loaded. The file \<project location\>\Sentry\Scripts\upload-debug-symbols-win.ps1 is
ubt> not digitally signed. You cannot run this script on the current system. For more information about running scripts and
ubt> setting execution policy, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
ubt>     + CategoryInfo          : SecurityError: (:) [], ParentContainsErrorRecordException
ubt>     + FullyQualifiedErrorId : UnauthorizedAccess
```

**Comment by riot-pmaconi** [comment]
@tustanivsky Windows Powershell (which is executed via `powershell` and what is specified in Sentry.uplugin) stops at version 5.1. Is the plugin intended to call into Powershell Core (which is executed via `pwsh`)? 

Here's an article explaining the binary name change:
https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.3

**Comment by tustanivsky** [comment]
> @tustanivsky Windows PowerShell (which is executed via `powershell` and what is specified in Sentry.uplugin) stops at version 5.1. Is the plugin intended to call into PowerShell (which is executed via `pwsh`)?
> 
> Here's an article explaining the binary name change: https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.3

I've made a quick check here and apparently both `powershell` and `pwsh` worked for me just fine (symbols were uploaded without any errors).

You can try to replace the current powershell command with the following and give it another try:
```
"pwsh \"$(PluginDir)/Scripts/upload-debug-symbols-win.ps1\" $(TargetPlatform) $(TargetName) $(TargetType) \"$(ProjectDir)\" \"$(PluginDir)\""
```

**Comment by riot-pmaconi** [comment]
Neither pwsh nor powershell is functional for me when included in PostBuildSteps. I'm only able to use the plugin if I erase it entirely.

**Comment by riot-pmaconi** [comment]
The execution policy list for powershell on this machine is:

```powershell
        Scope ExecutionPolicy
        ----- ---------------
MachinePolicy    RemoteSigned
   UserPolicy       Undefined
      Process          Bypass
  CurrentUser       Undefined
 LocalMachine       Undefined
```

**Comment by tustanivsky** [comment]
I see that your `MachinePolicy` is set to `RemoteSigned` which could be another reason why the system blocks the ps1 script downloaded from the internet. You can try to run `Unblock-File -Path <PATH_TO_PS1_SCRIPT>` and give it another go.

Also, it might be worth trying to experiment with different execution policies settings like this:
```
Set-ExecutionPolicy -Scope MachinePolicy -ExecutionPolicy Bypass
```

**Comment by tustanivsky** [comment]
Cant's say for sure atm, however I'll check some alternative mechanisms we can use for symbol uploading on Windows to avoid signing issues.

**Comment by bitsandfoxes** [comment]
Does https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_signing?view=powershell-7.3#running-unsigned-scripts-using-the-remotesigned-execution-policy unblock you?

**Comment by riot-pmaconi** [comment]
@bitsandfoxes Yes, manually unblocking the file on this machine allows Unreal to build. It's just not something I can scale out to the rest of my organization. 

For now I've just removed the step entirely as part of my integration. The automated symbol upload flow is not something I require, though it'd be ideal if I didn't need to make a note to prune the post build steps every time we upgrade the plugin.

**Comment by tustanivsky** [comment]
This is supposed to be addressed in #290. Basically, the `.ps1` was replaced with a similar `.bat` that doesn't depend on script execution policy of the machine it runs on.
